website/docs/d/netapp_volume_quota_rule.html.markdown

```diff
@@ -0,0 +1,61 @@
+---
+subcategory: "NetApp"
+layout: "azurerm"
+page_title: "Azure Resource Manager: Data Source: azurerm_netapp_volume_quota_rule"
+description: |-
+  Gets information about an existing Volume Quota Rule.
+---
+
+# Data Source: azurerm_netapp_volume_quota_rule
+
+Use this data source to access information about an existing Volume Quota Rule.
+
+## Example Usage
+
+```hcl
+data "azurerm_netapp_volume_quota_rule" "example" {
+  name                = "existing"
```

Comment:
Instead of `existing`, can we flesh these attributes out with something more in line with what you would pass into this datasource? 
```suggestion
  name                = "exampleQuotaRule"
```